client/admin/components/users/ImportDialog.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <v-dialog v-hotkey="{ esc: close }" v-model="visible" width="700">
+    <v-btn slot="activator" color="blue-grey" outline>
+      <v-icon>cloud_upload</v-icon>Import
+    </v-btn>
+    <v-form @submit.prevent="save">
+      <v-card class="pa-3">
+        <v-card-title class="headline">
+          Import Users
+        </v-card-title>
+        <v-card-text>
+          <v-text-field
+            ref="textFile"
+            v-model="filename"
+            :error-messages="showErrors"
+            :disabled="importing"
+            @click="$refs.fileInput.click()"
```

Comment:
Don't use javascript for this. Instead wrap both `<v-text-field>` and `<input type="file">` with `<label>` element. Click on label will activate inner input :bowtie: